modules/core/src/main/resources/scalafix-migrations.conf

```diff
@@ -60,8 +60,8 @@ migrations = [
     groupId: "org.typelevel",
     artifactIds: ["cats-core"],
     newVersion: "2.2.0-RC4",
```

Comment:
Should this be updated to `"2.2.0"`?

Reply:
The current value for `newVersion` would trigger the execution of the migration whenever the new version is **bigger** than `2.2.0-RC4`.As tried out by @fthomas this would bump users from `2.2.0-RC3` to `2.2.0-RC4` using the migration.

Projects on `2.1.1` would get upgraded to `2.2.0` using the migration once it is available without further action required. And if there would happen a `2.2.0-RC5` before users already on `2.2.0-RC4` wouldn't get the migration 2nd time.

Reply:
@mzuehlke is correct. - The only downside of not updating `newVersion` is that projects, that were already on `2.2.0-RC4` before the migration was added here, won't get the migration with the bump to `2.2.0`.

Since it seems that I'm the only one who tested the migration with the public Scala Steward instance so far (see https://github.com/search?q=author%3Ascala-steward+is%3Apr+cats+2.2.0-RC4+Applied+Migrations) it might make sense to bump `newVersion` to `2.2.0` nonetheless. I don't mind if Scala Steward applies the migration a 2nd time on my projects.

Reply:
https://github.com/scala-steward-org/scala-steward/pull/1596

Reply:
👍